tests/apichecks/crds_test.go

```diff
@@ -390,3 +397,151 @@ func TestCRDShortNames(t *testing.T) {
 	want := strings.Join(errs, "\n")
 	test.CompareGoldenFile(t, "testdata/exceptions/shortnames.txt", want)
 }
+
+// Run this test with WRITE_GOLDEN_OUTPUT set to update the expections list.
+// Note adding to the exceptions list should not be done trivially.
```

Comment:
If you're fixing the spelling mistake here, maybe "adding to the exceptions list should primarily be used for backwards compatibility" ?

Reply:
Hmmm, there are a lot of them, so maybe just drop the note :-)